lib/shrine/plugins/logging.rb

```diff
@@ -68,6 +74,7 @@ def logger
             logger.level = Logger::INFO
             logger.level = Logger::WARN if ENV["RACK_ENV"] == "test"
             logger.formatter = pretty_formatter
+            opts[:logging_logger] = logger
```

Comment:
I would prefer if `Shrine.opts` is modified only when loading plugins (in the `configure` method), I think that modifying it dynamically like that when invoking reader methods could potentially lead so subtle bugs.

I was thinking that we could move all this code for creating the default logger into the `configure` method.

Reply:
Agree. I'll make the changes.